cli-feed-v3.json

```diff
@@ -104,24 +104,14 @@
       "sha2": "A148DC9C2783D2E7E5E50F59BE2EAEA7948213A30876F41171CF20E9F84754F2",
       "FUNCTIONS_EXTENSION_VERSION": "beta"
     },
-    "2.2.0": {
-      "Microsoft.NET.Sdk.Functions": "1.0.13",
-      "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.30/Azure.Functions.Cli.win-x86.2.0.1-beta.30.zip",
-      "localEntryPoint": "func.exe",
-      "itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-beta-10202",
-      "projectTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Microsoft.AzureFunctions.ProjectTemplates/2.0.0-beta-10202",
-      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/2.0.0-beta-10183.zip",
-      "sha2": "DC1D0F2D2B5817E27B78CF5109EA086F865D76867ADD3FD510287CA92754FFE8",
-      "FUNCTIONS_EXTENSION_VERSION": "beta"
-    },
-    "2.2.1": {
-      "Microsoft.NET.Sdk.Functions": "1.0.13",
+    "2.2.2": {
+      "Microsoft.NET.Sdk.Functions": "1.0.14",
       "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.30/Azure.Functions.Cli.win-x86.2.0.1-beta.30.zip",
       "localEntryPoint": "func.exe",
       "itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-beta-10205",
       "projectTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Microsoft.AzureFunctions.ProjectTemplates/2.0.0-beta-10205",

```

Comment:
Need updated project template here that references Microsoft.NET.Sdk.Functions 1.0.14

Reply:
okay, put the link here and I'll update the PR

Reply:
```
"itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-beta-10210",
"projectTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Microsoft.AzureFunctions.ProjectTemplates/2.0.0-beta-10210"

```